README.md

```diff
@@ -1,20 +1,20 @@
-# Project Name
+# Odin-youtube-clone
```

Comment:
Let's dd the project name without using snake-case. I think `Embedding videos and images: Youtube clone` is better.

Reply:
Nice suggestion!, Thanks